oda_api/api.py

```diff
@@ -853,9 +862,10 @@ def get_product(self,
                 'parameter check not available on remote server, check carefully parameters name')
         else:
             _ignore_list = ['instrument', 'product_type', 'query_type',
-                            'off_line', 'query_status', 'verbose', 'session_id', 'dry_run']
+                            'off_line', 'query_status', 'verbose', 'session_id']
```

Comment:
so now there will be two warnings? And it will be sent, and will actually affect the analysis since dispatcher still supports it?

Reply:
You mean one from here and one from the dispatcher?

Reply:
I mean in validation in the next lines.

Reply:
In any case, depends on what we want to do: we can still have it ignored by the oda_api, as it was before, with the new warning saying that the parameter has no effect, then we remove it from the dispatcher.

Otherwise, for the moment, we can warn the user that the support for this parameter will end soon, but we still support it for the moment.

I personally would leave the warning we introduced now and remove the support from the dispatcher.

Reply:
> In any case, depends on what we want to do: we can still have it ignored by the oda_api, as it was before, with the new warning saying that the parameter has no effect, then we remove it from the dispatcher.
> 
> Otherwise, for the moment, we can warn the user that the support for this parameter will end soon, but we still support it for the moment.
> 
> I personally would leave the warning we introduced now and remove the support from the dispatcher.
> 
> But of course, I have no experience regarding the usage of such a parameter.

You can just remove it from `kwargs` right after you warn about it not being used. 
This will prevent an additional warning in parameter validation in oda_api, and also avoid any confusion about what is done by the dispatcher.

Reply:
I agree, it is the smoothest approach.